### PR TITLE
Update Readme and add license

### PR DESCRIPTION
**Pull request by blaketarter**
## Description

Updated the readme to changes references to crema-app-web and update parts that are specific to the mobile repo.

Related to #5 

## 🔎 Reviewer Checklist

> Checked off by the PR **Reviewers**

### Required

> These always need to be checked

- [x] Merge destination is correct
- [x] Code is correct as understood and conforms to quality standards
- [x] Tests have been added where appropriate (unit, visual, end-to-end)
- [x] Acceptance Criteria have been met

---

>### Roles & Responsibilities
>
>#### 👨‍💻 Assignee
>
>- Initiator of this PR (be sure to set in GitHub UI)
>- Addresses feedback and change requests
>- Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
>#### 👩‍💻 Reviewer
>
>- Invited to review PR by **Assignee** (via GitHub UI)
>- Is expected to complete a review and address followup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,28 +37,24 @@ This project includes configuration and tooling that conforms to Crema's baselin
...
-- `test:analyze` - run bundle analyzer
-- `test:deps` - run dependency validation tests
-- `test:e2e` - run end-to-end tests
-- `test:lint:fix` - run linter and fix if possible
+- `start` - start the expo bundler
+- `start:ios` - start the expo bundler and an iOS simulator
+- `start:android` - start the expo bundler and an android emulator
+- `start:web` - start the expo bundler and the web app
+- `start:playground` - start component playground with storybook
+- `start:types` - start tsc in watch mode
```
Comment (roblafeve) [inline README.md:50]: Does this need to be run manually?

File: README.md [hunk 2]
```diff
@@ -41,28 +37,24 @@ This project includes configuration and tooling that conforms to Crema's baselin
...
-- `new:util` - generate a new util(ity)
-- `test:analyze` - run bundle analyzer
-- `test:deps` - run dependency validation tests
-- `test:e2e` - run end-to-end tests
-- `test:lint:fix` - run linter and fix if possible
+- `start` - start the expo bundler
+- `start:ios` - start the expo bundler and an iOS simulator
+- `start:android` - start the expo bundler and an android emulator
+- `start:web` - start the expo bundler and the web app
+- `start:playground` - start component playground with storybook
```
Comment (roblafeve) [inline README.md]:
```suggestion
- `start:visual` - start component playground with storybook
```
🤷‍♂

**Review by roblafeve — APPROVED** [review]
🙆‍♂

File: README.md (hunk 1 above)
Comment (blaketarter) [reply README.md:50]: @roblafeve Yes, the expo process does not display type checks before or during bundling. 😞

File: README.md (hunk 2 above)
Comment (blaketarter) [reply README.md]: @roblafeve This command doesn't start the visual tests, it starts the app and opens up the Storybook UI.

File: README.md (hunk 1 above)
Comment (roblafeve) [reply README.md:50]: Booo. Oh well 🤷‍♂

File: README.md (hunk 2 above)
Comment (roblafeve) [reply README.md]: Right, I was wondering if having a `start:visual` that corresponds with `test:visual` would make sense. It kind of conveys a relationship between them. Something I've wondered about on the web side too. Food for 💭 .

Comment (blaketarter) [reply README.md]: Hmmm, that makes sense actually. 🤔

Comment (roblafeve) [reply README.md]: 🤷‍♂

**Pull request merged**